src/commands/apphosting-secrets-set.ts

```diff
@@ -63,17 +63,12 @@ export const command = new Command("apphosting:secrets:set <secretName>")
       secretValue = fs.readFileSync(dataFile, "utf-8");
     }
 
-    if (created) {
-      logSuccess(`Created new secret projects/${projectId}/secrets/${secretName}`);
-    }
-
     const version = await gcsm.addVersion(projectId, secretName, secretValue);
     logSuccess(`Created new secret version ${gcsm.toSecretVersionResourceName(version)}`);
-    logSuccess(howToAccess);
+    logBullet(howToAccess);
 
     // If the secret already exists, we want to exit once the new version is added
     if (!created) {
-      logWarning(grantAccess);
```

Comment:
Instead of logging this when the secret is not created. We should try to check if the service accounts already have access. If they don't then we should log. 

When the user creates a new secret but declines to grant access to the service accounts then following versions of the secret will not have access either.

Reply:
@tonyjhuang let me know what you think?

Reply:
IMO this current flow seems reasonable to me. If the secret previously existed and the user is adding a new version, it seems appropriate to expect the secret is already correctly provisioned and we can exit out early